torch/csrc/StorageSharing.cpp

```diff
@@ -66,9 +66,8 @@ static PyObject* THPStorage_pyNewFilenameStorage(
     PyObject* _unused,
     PyObject* args) {
   HANDLE_TH_ERRORS
-  // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-  long long size;
-  if (!PyArg_ParseTuple(args, "L", &size)) {
+  unsigned long long size = 0;
+  if (!PyArg_ParseTuple(args, "K", &size)) {
```

Comment:
Can you add a test that validates, that passing negative value here will raise an error of sorts?

Reply:
We can revert to use long long and check negative